weld.fs.js

```diff
@@ -961,7 +1015,7 @@ function sketchVButtWeld(context is Context, definition is map, thickness is Val
     var rootGapWidth = definition.buttRootGapWidth;
     var rootGapHeight = definition.buttRootGapHeight;
 
-    var distOut = rootGap ? tan(angle / 2) * (thickness - rootGapHeight) + rootGapWidth / 2 : tan(angle / 2) * thickness;
+    var distOut = rootGap ? tan(angle) * (thickness - rootGapHeight) + rootGapWidth / 2 : tan(angle) * thickness;
```

Comment:
@emagdalenaC2i 
Many of the websites I have looked at for how the angle of a V-Butt weld is measured show the angle as the angle between the cut faces.

I would suggest changing this back to `var distOut = rootGap ? tan(angle / 2) * (thickness - rootGapHeight) + rootGapWidth / 2 : tan(angle / 2) * thickness;` unless you want to add an option for bevel angle in the feature.

Websites:
http://triblab.teipir.gr/files/Welding/Lab/CH3_2_Welding_joint_preparation.pdf#page=13
https://www.catiadesign.org/_doc/v5r14/catpdfwd1ug_C2/wd1ug.pdf#page=60
https://www.haynesintl.com/alloys/fabrication-brochure/welding-and-joining/weld-joint-design
https://www.twi-global.com/technical-knowledge/job-knowledge/design-part-3-092

Reply:
This is just to keep a standard format entry for the angle. If not the ANGLE_STRICT_90_BOUNDS doesn' work fine with V_BUTT_WELD and U_BUTT_WELD and we should change again to ANGLE_STRICT_180_BOUNDS
@mbartlett21 I propose this change because the angle in V_BUTT_WELD is 2 times the angle in BEVEL_BUTT_WELD, and the angle in U_BUTT_WELD is 2 times the angle in J_BUTT_WELD.
With this change the user always insert the same angle

Reply:
@mbartlett21 You said "unless you want to add an option for bevel angle in the feature."
Of course, that option is already set

Reply:
Can you maybe change the UI annotation to say "Bevel angle" or something like that?

Reply:
@mbartlett21 done